Blazor/Components/ChangeLang.razor

```diff
@@ -0,0 +1,25 @@
+﻿@using  System.Globalization
+@inject IJSRuntime JS
+@inject NavigationManager Navigation
+@page "/ChangeLang"
+
+@code {
+    public static string currentPage;
+
+    protected override async Task OnInitializedAsync()
+    {
+        await JS.InvokeVoidAsync("GetCurrentPage");
+        var js = (IJSInProcessRuntime)JS;
```

Comment:
Just curious about the typecast: 
- Are there other types of `JS` that it could be? (for example could user use another runtime in their app and cause us to crash?)   In other words should be handle other possible sub-classes here?
- Should we do a "safe" typecast here (with `as`) and just not do the `InvokeVoid` is null
- OR should (is it even possible?) we wrap "blazorCulture.set" in an async function and get rid of the typecast altogether?

Reply:
Fixed - no typecast required afterall